tempto-examples/src/main/resources/tempto-configuration.yaml

```diff
@@ -77,6 +77,8 @@ databases:
 
 tests:
   fs:
+    type: hdfs
```

Comment:
I do not see any usages of this.

Reply:
The usage of this will be visible when I add the S3 client changes on top of these commits.  Should I add that in the same PR?  I have the changes already and just need to finish some testing.